app/models/institution_builder.rb

```diff
@@ -558,4 +559,33 @@ def self.build_institution_programs(version_number)
     sql = InstitutionProgram.send(:sanitize_sql, [str, version_number])
     InstitutionProgram.connection.execute(sql)
   end
+
+  def self.build_versioned_school_certifying_official(version_number)
+    str = <<-SQL
+    INSERT INTO versioned_school_certifying_officials(facility_code,
```

Comment:
Can we clean up the indentation/line breaks of this SQL?
See `build_institution_programs` for a good example.